OpenRA.Mods.Common/Traits/Buildings/TransformsIntoDockClientManager.cs

```diff
@@ -66,12 +67,18 @@ IEnumerable<IOrderTargeter> IIssueOrder.Orders
 			get
 			{
 				yield return new DockActorTargeter(
-					6,
-					Info.EnterCursor,
-					Info.EnterBlockedCursor,
-					() => Info.RequiresForceMove,
-					CanQueueDockAt,
-					CanDockAt);
+					priority: 6,
+					context =>
+					{
+						if (Info.RequiresForceMove && !context.ForceEnter)
+							return CanTargetResult.Blocked(Info.EnterCursor);
```

Comment:
```suggestion
							return CanTargetResult.Blocked();
```
why pass values that will be null?

Reply:
Why null? I made the refactored logic pick the same with the cursor as the original logic. The way it's written now allows reducing number of ifs and it's easy to follow. Do you mean something else?

Reply:
The constrictor sets it to null. The value you input is ignored

Reply:
You're right, my bad. Fixed.